lib/razor/config.rb

```diff
@@ -43,7 +43,7 @@ def initialize(env, fname = nil, defaults_file = nil)
       # Use the filename given, or from the environment, or from /etc if it
       # exists, otherwise the one in our root directory...
       fname ||= ENV["RAZOR_CONFIG"] ||
-        (File.file?('/etc/razor/config.yaml') and '/etc/razor/config.yaml') ||
+        (File.file?('/etc/puppetlabs/razor-server/config.yaml') and '/etc/puppetlabs/razor-server/config.yaml') ||
```

Comment:
I don't think this PR should be blocked on that decision, since it is a packaging change. What do you think, @smcclellan ?